scripts/ci/requirements.txt

```diff
@@ -88,7 +88,7 @@ Sphinx-PyPI-upload
 sphinx_rtd_theme
 sqlalchemy
 statsd
-thrift
+thrift==0.9.3
```

Comment:
Airflow requires 0.9.2+

Reply:
impyla requires <= 0.9.3

Reply:
we will never use impyla, but it is required for the [Hive hooks](https://github.com/apache/incubator-airflow/blob/master/airflow/hooks/hive_hooks.py)